### DISCUSSION TITLE
Welcome to WRF-MASTER Discussions!

### DISCUSSION THREAD

**Discussion by HathewayWill** [HathewayWill/WRF-MOSIT#1] [Announcements]
.

**Comment by Pops-CS** [comment]
![image](https://github.com/user-attachments/assets/61369355-9269-443a-b773-d806d1101053)
why can not click on this page anymore

**Reply by HathewayWill** [reply]
when did you download the code?

**Reply by Pops-CS** [reply]
It went down at 10:00 on September 9th

**Reply by HathewayWill** [reply]
which operating system are you using?

**Comment by Pops-CS** [comment]
[屏幕录制 2024-09-10 093629.zip](https://github.com/user-attachments/files/16938207/2024-09-10.093629.zip) I videotaped the screen and compressed it into a zip pack. Could you please take a look at the whole process? Thank you very much

**Reply by HathewayWill** [reply]
I'm familiar with this issue, what operating system are you using.

**Comment by selvetikarashok** [comment]
Dear Sir,

Thank you very much for providing the installation script for WRF-MOSIT. The installation process went smoothly overall, and I appreciate your guidance.

However, I have encountered a few errors during the installation on my desktop, as shown in the attached log file. I was attempting to set up WRF-Chem using WRF-MOSIT, and I would like to request your kind assistance in verifying whether the installation was completed successfully or if any further steps are required.

Thank you once again for your valuable support and time. I look forward to your suggestions.

[WRF_MOSIT.log](https://github.com/user-attachments/files/23390484/WRF_MOSIT.log)

**Reply by HathewayWill** [reply]
okay there seems to be an issue with the code not stopping when it's supposed to.

What options did you pick to install wrf chem?

GNU? Intel?
autoconfig?
etc etc
